daemon/src/maker_cfd.rs

```diff
@@ -478,13 +479,30 @@ impl<O, T, W> Actor<O, T, W> {
 
         let order_id = msg.order_id;
 
+        let mut conn = self.db.acquire().await?;
+        let cfd = load_cfd(order_id, &mut conn).await?;
+        let funding_rate = match cfd.position() {
+            Position::Long => {
+                current_offers
+                    .long
+                    .context("Long funding rate not available")?
+                    .funding_rate
+            }
+            Position::Short => {
+                current_offers
+                    .short
+                    .context("Short funding rate not available")?
+                    .funding_rate
+            }
+        };
+
```

Comment:
IMHO these details should not go from a particular order. you can have order `None`, but still apply funding rate, allowing already established Cfds to keep rolling over.

Reply:
as per conversations in element, we moved to having 2 global funding rates (long and short). I'll remove the daily_funding_rate from within `Order` in a follow-up commit.